index.html

```diff
@@ -50,5 +50,5 @@ <h3>Contact information</h3>
     <hr>
     <p>All right reserved&copy;</p>
   </body>
-  <script src="../dist/final.js"></script>
+  <script src="./index.js"></script>
```

Comment:
- [ ] Great effort working on the Awesome book project `Refactor code using ES6` milestone. Currently, I noticed that while implementing the arrow functions, the `remove` book button has been tampered with and I find it difficult to remove whatever book I have added. Kindly fix it and be sure all the necessary functionalities are 

Video presentation of the claim 👇 

[screen-capture.webm](https://github.com/Shahid-Bagwan/AwesomeBook/assets/78634841/8bd7f663-f2d4-446e-b606-1919dc20d5f7)

Reply:
solved @brainconnect93